### DISCUSSION TITLE
Playback screen UI improvements

### DISCUSSION THREAD

**Discussion by dotancohen** [PaulWoitaschek/Voice#1898] [Ideas]
Some common actions require redundant clicks on the playback screen. These actions should have large icons that are easily accessed on the playback screen.

For instance, adding a bookmark requires one to first open the bookmarks window, then click the + icon, then choose a name, then return to the playback screen. In my opinion, as one who uses the bookmark feature to mark a place to reference later, there should be a single button on the playback screen that adds a bookmark with that one click.

Another example is the playback speed control. Currently, one has to click the playback speed button, move his hand to the popup slider, then tap to close the slider. In my opinion, the button should be much larger and in the middle or center of the screen, such that upon click a slider appears under the user's finger, which can then slide up/down or left/right without being lifted from its original click location. Upon lifting the finger, the slider should close.

**Describe the solution you'd like**
Larger icons with direct access to commonly-used features.

**Describe alternatives you've considered**
Clicking and mis-clicking too often!

**Additional context**
I argue that the features mentioned are already exposed on the playback screen, thus it is expected that they would be commonly-used features. I'm suggesting how to make them better.

**Comment by drzraf** [comment]
Some more comments on this after having tried this app' with a blind person. The "speed" popup relies on clicking outside the popup to close it. But with talkback on, there is no way this popup can be closed (it has no "close" handle). This is a blocking point because it's too easy to make it appear and impossible to close.

For an audiobook app', I suggest to try using the app' after activating `talkback` (and closing the eyes) to realize some of the current UI limitations

**Reply by drzraf** [reply]
(friendly ping)  [not sure whether, @PaulWoitaschek, you're reading older discussions]
